chef/cookbooks/neutron/recipes/post_install_conf.rb

```diff
@@ -61,26 +62,35 @@ def mask_to_bits(mask)
 end
 neutron_cmd = "neutron #{neutron_args}"
 
+fixed_network_type = ""
 floating_network_type = ""
-if node[:neutron][:networking_mode] == 'vlan'
-  fixed_network_type = "--provider:network_type vlan --provider:segmentation_id #{fixed_net["vlan"]} --provider:physical_network physnet1"
-  if node[:network][:networks][:nova_floating][:use_vlan]
-    floating_network_type = "--provider:network_type vlan --provider:segmentation_id #{floating_net["vlan"]} --provider:physical_network physnet1"
-  else
+
+# TODO(toabctl): For now we just use the first ml2_type_driver for foating network type
```

Comment:
> "foating"

Done in https://github.com/crowbar/barclamp-neutron/commit/6573fd00800313318448834fb6177af75676afa8